.zuul.d/jobs.yaml

```diff
@@ -0,0 +1,14 @@
+- job:
+    name: pypa-pip-base
+    abstract: true
+    description: Run pypa/pip unit testing
```

Comment:
It's not just unit testing so maybe this should be renamed
```suggestion
    description: Run pypa/pip test suite
```

Reply:
done

Reply:
@ianw It still says "units" here